modules/qotd/schedule.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+import asyncio
+from disnake.ext.commands.bot import InteractionBot
+from disnake import ForumChannel, Embed, Colour
+from variables import qotd_channel, qotd_input_channel, qotd_role
+
+def _seconds_until_oclock(hour: int):
+    now = datetime.datetime.now()
+    target = now.replace(hour=hour, minute=0, second=0, microsecond=0)
+    if now > target:
+        target += datetime.timedelta(days=1)
+    return (target - now).total_seconds()
+
+async def schedule_qotd(bot: InteractionBot):
+    while True:
+        await asyncio.sleep(_seconds_until_oclock(18))
+        # Everything past here will only be execute once a day at 18:00 (I don't know the timezone though)
+
+        # Find QOTD template to post
+        input_channel = bot.get_channel(qotd_input_channel) #! replace with actual QOTD input channel
+        template_message = None
+        async for message in input_channel.history(limit=200):
+            if '✅' in message.reactions or type(message.content) != str:
+                continue
+            template_message = message
+            await message.add_reaction('✅')
+            break
+
+        template_text = str(template_message.content).split('\n')
```

Comment:
Maybe add a check here to see if the message is still `None`, and send a message in the channel to say that we've run out of QOTDs (and I imagine field access on `None` causes an exception)